frontend/client/components/CreateFlow/index.tsx

```diff
@@ -174,12 +174,13 @@ class CreateFlow extends React.Component<Props, State> {
           <div className="CreateFlow-header">
             <Steps current={currentIndex}>
               {STEP_ORDER.slice(0, 5).map(s => (
-                <Steps.Step
+                <div
```

Comment:
Something around here might be the reason the steps are missing for me, and I'm seeing the console warning (see my [comment](https://github.com/grant-project/zcash-grant-system/pull/131#issuecomment-459787346)).

Reply:
Yep, that does explain it. For some reason they removed the `onClick` prop so I tried to work around it, dunno why it was working for me earlier. I'll just `any` type it to get around this issue and open a new issue on their repo.